src/extension.ts

```diff
@@ -74,20 +74,13 @@ type MachineJSONListOutput = {
   error: string;
 };
 
-export let macadam: Macadam;
+export let macadam: macadamJSPackage.Macadam;
 
 export const macadamMachinesStatuses = new Map<string, extensionApi.ProviderConnectionStatus>();
 
 export async function activate(extensionContext: extensionApi.ExtensionContext): Promise<void> {
-  macadam = new Macadam(extensionContext.storagePath);
-
-  let binary: BinaryInfo | undefined = undefined;
-  // retrieve macadam
-  try {
-    binary = await macadam.getBinaryInfo();
-  } catch (err: unknown) {
-    console.error(err);
-  }
+  macadam = new macadamJSPackage.Macadam('RHEL extension');
```

Comment:
The parameter is for the moment not used by the macadam.js library, but it is expected that next change will be to use this paramter's value as part of the VM name. It would be more reasonable to have something simpler, and without spaces (`rhel`)